cloud/logger/src/index.ts

```diff
@@ -4,3 +4,23 @@ import {ConsoleLogger} from './ConsoleLogger';
 export {Logger} from  './Logger';
 export {BunyanLogger} from  './BunyanLogger';
 export {ConsoleLogger} from './ConsoleLogger';
+
+
+import {Logger} from  './Logger';
+
+// Global logger that will be used in all applications
+export let logger: Logger; 
```

Comment:
shouldnt this be const since its global and intended to live alongside application?